src/search/landmarks/landmark_status_manager.cc

```diff
@@ -71,6 +66,11 @@ LandmarkStatusManager::LandmarkStatusManager(
           progress_reasonable_orderings
           ? get_reasonable_parents(graph)
           : vector<pair<LandmarkNode *, vector<LandmarkNode *>>>{}),
+      /*
+        By default we mark all landmarks past, since it is the neutral element
+        for set intersection which we emulate in the progression. Similarly, no
+        landmarks are future which is the neutral element for set union.
```

Comment:
I think something got mangled at some point in this issue, or perhaps a previous issue (not in this commit). I think these sentences don't make sense any more, the "it" should reference the words "true" and "false", but they are not there any more. It is not clear to me what the "it" is supposed to mean now in these two sentences.

Reply:
Suggestion: `We initialize to true in past_landmarks because true is the neutral element of conjunction/set intersection. We initialize to false in future_landmarks because false is neutral for disjunction/set union.`